inventory/dynamic/azure/ansible.cfg

```diff
@@ -0,0 +1,45 @@
+# config file for ansible -- http://ansible.com/
```

Comment:
is this identical to the gcp file? any sense in symlinking it?

Reply:
I think only the `private_key_file` option is different.